### ISSUE TITLE
not all bind function got converted to the arrow functions

### ISSUE THREAD

**Issue by immortal-loop** [markalfred/eslint-plugin-no-constructor-bind#42]
thanks, Mark, I am enjoying this plugin. However, I have got another issue.

As shown in the screenshot below after running the fix
1. not all bind function got converted to the arrow functions.
2. line 57 and 58 there are semicolons left.

file data

```

class Feature extends React.Component {
    constructor(props) {
        super(props);

        this.state = {
            header: null,
            compactVMenu: false,
            renderPause: false
        }

        this.headerUpdateTimer = null;
        this.disableAnimations = false;
        this.vMenuRef = null;
        this.vMenuFocusId = 'vMenu';

        this.updateHeader = this.updateHeader.bind(this);
        this.changeFocusTo = this.changeFocusTo.bind(this);
        this.menuClickHandler = this.menuClickHandler.bind(this);
        this.assetClickHandler = this.assetClickHandler.bind(this);
        this.onMenuBlur = this.onMenuBlur.bind(this);
        this.onMenuFocus = this.onMenuFocus.bind(this);
        this.setVMenuRef = this.setVMenuRef.bind(this);
        this.handleDocumentKeyDown = this.handleDocumentKeyDown.bind(this);
        this.forceFocusMenu = this.forceFocusMenu.bind(this);
        this.selectedNonFirstTrayItem = this.selectedNonFirstTrayItem.bind(this);
        this.selectedFirstTrayItem = this.selectedFirstTrayItem.bind(this);
        this.swapToKidsMenu = this.swapToKidsMenu.bind(this);
        this.swapToHomeMenu = this.swapToHomeMenu.bind(this);
        this.resumeRender = this.resumeRender.bind(this);
        this.pauseRender = this.pauseRender.bind(this);
        this.loadTraysForSubMenu = this.loadTraysForSubMenu.bind(this);
        this.forceRelaod = this.forceRelaod.bind(this);
        this.routeEnded = this.routeEnded.bind(this);
        this.fireAppStartEvent = this.fireAppStartEvent.bind(this);

        document.addEventListener('keydown', this.handleDocumentKeyDown);
        props.actions.showLoading();
    }

    componentDidMount() {
    }

    componentDidUpdate(prevProps) {
    }

    componentWillUnmount() {
    }

    updateVerticalHeight() {
    }

    async forceRelaod(){}

    changeFocusTo(index, tray) {
    }

    routeEnded(){
    }

    fireAppStartEvent () {
    }

    onMenuFocus() {
    }

    onMenuBlur() {
    }

    updateHeader(selectedItem) {
    }

    menuClickHandler(menuItem) {
    }

    async swapToKidsMenu ({ id }) {
    }

    async swapToHomeMenu () {
    }

    loadTraysForSubMenu() {
    }

    pauseRender () {
    }

    resumeRender () {
    }

    assetClickHandler(asset) {
    }

    handleDocumentKeyDown(event) {
    }

    handleBackPressed() {
    }

    forceFocusMenu(noDelay) {
    }

    setVMenuRef(verticalMenu) {
    }

    showExitDialog() {
    }

    onBlurLists() {
    }

    onFocusLists() {
    }

    selectedNonFirstTrayItem() {
    }

    selectedFirstTrayItem() {
    }

    render() {
    }
}

```

<img width="1671" alt="Screenshot 2020-06-17 at 6 12 54 PM" src="https://user-images.githubusercontent.com/66246267/84899343-2ccf0780-b0c6-11ea-8890-5ad4067b955a.png">

**Comment by markalfred** [comment]
Looks like there's an issue handling semicolons, thank you @immortal-loop I'll get to this ASAP 👍

**Comment by immortal-loop** [comment]
thanks @markalfred 
looking forward to it

**Comment by markalfred** [comment]
@immortal-loop this should be fixed in v2.0.2, but I noticed something when testing against your code: it will fix a bunch of the functions when you run `eslint --fix`, but it doesn't fix all of them. Instead, it says `10 errors and 0 warnings potentially fixable with the `--fix` option.` If you run `eslint --fix` again, *then* it'll fix those remaining 10.

Thanks for opening the issue and providing your source code, it was very helpful. 🙇

**Comment by vikas-bansal-DisneyPH** [comment]
Thanks @markalfred for investing your time brother
